src/worker.ts

```diff
@@ -52,38 +36,20 @@ export class Worker {
           `-p`,
           `${this.port}:8000`,
           "--cpus",
-          "0.2",
+          "0.1",
           "--memory",
-          "200m",
+          "30m",
           "worker:latest",
         ],
       },
     ).spawn();
-    this.#running = true;
-    this.#emit("listening", { id: this.name, port: this.port });
-
-    this.#process.status.then(({ code }) => {
-      console.log(
-        "[worker]",
-        `${this.name}::module(${this.module?.name}@${this.module?.version})`,
-        "stopped with",
-        code,
-      );
-      this.#running = false;
-      this.#emit("shutdown", { code });
-    });
-  }
```

Comment:
I'm trying to think through if this is an issue. Not being informed when the process fails or ends unexpectedly seems like it may be an issue.